### DISCUSSION TITLE
Weather indication not working

### DISCUSSION THREAD

**Discussion by Dominiquini** [rocksdanister/lively#1458] [General]
**Describe the bug**

**To Reproduce**
1. Open Lively
2. Select "Ripples"
3. Customize the wallpaper to show the weather
4. Insert the city code and OpenWeather API
5. View message to enter api/city information

**Expected behavior**
I should see the weather displaying on the background

**Screenshots/Video**
![yOcwA30Qau](https://user-images.githubusercontent.com/1180808/201953202-0c9d056d-9c41-4c8b-a105-6c22a90704dc.jpg)

**Desktop (please complete the following information):**
 - OS: Windows 10 x64 19045.2251
 - Lively: 2.0.5.0
 - "Ripples"

**Additional context**
I tried this wallpaper too (https://github.com/rocksdanister/weather-fetch-wallpaper) and the same problem occurs!

**Log file (Important)**
[20221115_130603.txt](https://github.com/rocksdanister/lively/files/10014261/20221115_130603.txt)

**Comment by rocksdanister** [comment]
If you recently created the API key then it takes time to activate on openweathermap side.

**Comment by Dominiquini** [comment]
> If you recently created the API key then it takes time to activate on openweathermap side.

I tried my old API key bedore! When it not worked, I created a new one... None of then worked!

Thanks.

**Comment by rocksdanister** [comment]
<img width="460" alt="image" src="https://user-images.githubusercontent.com/17554161/202570009-f4f85c5b-fb11-424b-9ef4-9afb9d393ecc.png">

Tested working for me, enter location as `city, country`  example: `new delhi, india`

**Comment by xhohoho** [comment]
im having the same problem, when testing the sample code with generated api in browser... it return data. but still showing blank in lively
![image](https://user-images.githubusercontent.com/56391044/204230754-e2de206c-79ac-4ddf-9544-db7bf34f729a.png)
![image](https://user-images.githubusercontent.com/56391044/204230793-638ff0a5-73cd-4729-bc6a-05b1bb7a9456.png)

**Reply by xhohoho** [reply]
and also the api has been a month created
![image](https://user-images.githubusercontent.com/56391044/204231188-2535e063-72d9-4918-abb6-41f6198a63e2.png)

**Reply by rocksdanister** [reply]
@dabjulmaros

**Comment by rocksdanister** [comment]
Open the log window and try starting the wallpaper, see if there is any error:
<img width="917" alt="image" src="https://user-images.githubusercontent.com/17554161/204266359-97c81898-5ad3-4767-89b6-2610421af7e5.png">

**Reply by xhohoho** [reply]
![image](https://user-images.githubusercontent.com/56391044/204409625-aa90398c-8b44-4527-b45b-61c8988d4e5b.png)

Thanks for the quick reply. It is the same text as in log.

**Reply by lazy-boy-cup** [reply]
Can you update this wallpaper?please

**Comment by megaroeny** [comment]
What is the correct way to enter my city, if I live in USA? @rocksdanister 

- City, USA
OR
- City, State (like VA)

**Reply by rocksdanister** [reply]
Same input as: https://openweathermap.org/

**Reply by megaroeny** [reply]
Ahhh okay. Thank you!
